packages/svg-icons/tests/jest/validateRichIconSvgs.test.ts

```diff
@@ -57,6 +57,38 @@ describe("SVGs", () => {
                 expect(otherGroup).toStrictEqual(firstGroup);
             });
     });
+
```

Comment:
The documentation should be updated to say to add this in the contributing guidelines